coeus-impl/src/main/java/org/kuali/kra/irb/ProtocolDocument.java

```diff
@@ -163,6 +171,13 @@ protected void mergeAmendment(String protocolStatusCode, String type) {
         }
         try {
             getDocumentService().saveDocument(newProtocolDocument);
+            //manually set the principleID of the last actionrequest to BLANK, so this routes
+            List requests = getActionRequestService().findAllActionRequestsByDocumentId(newProtocolDocument.getDocumentHeader().getDocumentNumber());
+            if (requests != null && requests.size() > 0) {
+            	ActionRequestValue lastRequest = (ActionRequestValue)requests.get(requests.size() -1);
+            	lastRequest.setPrincipalId("");
+            	getDataObjectService().save(lastRequest);
+            }
             // blanket approve to make the new protocol document 'final'
```

Comment:
Why are we doing this? So there are action requests which we are invalidating? What are these action requests?